### DISCUSSION TITLE
Datetime picker and Datetime Range Picker

### DISCUSSION THREAD

**Discussion by Nikhilchandra-Rathod** [mantinedev/mantine#2900] [Feature requests]
How can I use datetime picker and datetime range picker ?

**Comment by rtivital** [comment]
You can install 6.0 dates package preview for date time picker – https://rtivital.github.io/mantine-dates-6/date-time-picker

**Reply by jukrb0x** [reply]
looks dope, but it's still required "input" instead of "pick"?
